### ISSUE TITLE
[Android] lint fails many incomplete translations

### ISSUE THREAD

**Issue by tenuki** [WhatsApp/stickers#734]
**Specifications**  
OS Version: win10
Device Manufacturer: -

**Problem Description**  
-project linting fails

**Reproduction Steps**  
gradlew build

**Head of Commit**
c91bab995ebc177e3de63c5bbb735ee258b60292

**Comment by simonzhexu** [comment]
yep, aware of that, thanks for filing the issue, will try to fix.

**Comment by kaneki666** [comment]
There is a problem with this sticker pack and it can't be added to whatsapp!! I am getting this error
